decko-rails/config/routes.rb

```diff
@@ -1,6 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-Rails.application.routes.draw do
+Decko::Engine.routes.draw do
```

Comment:
I actually thought this had been in for a while already.  This is actually what makes in mountable.